tests/common/bats/update-ips.bash

```diff
@@ -63,7 +63,16 @@ update_ips.mock_maximal() {
   # main swift
 
   # GET /auth/tokens
-  mock_set_output "${mock_curl}" '{"token":{"catalog":[{"type": "object-store", "name": "swift", "endpoints": [{"interface":"public", "region": "swift-region", "url": "https://swift.foo.dev-ck8s.com:91011"}]}]}}' 1
+  mock_set_output "${mock_curl}" 'HTTP/2 200
+  date: Wed, 09 Oct 2024 14:14:40 GMT
+  expires: -1
+  cache-control: private, max-age=0
+  content-type: text/html; charset=ISO-8859-1
+  x-subject-token: 123456789
+  accept-ranges: none
+  vary: Accept-Encoding
+  \r\n
```

Comment:
Will this not print literal '\r\n'? Although I would not expect the test to succeed if that was the case, hmmm...

Reply:
Ah, I think we might get away with it because we never test what is getting fed to dig or curl. Could you add some `mock_get_call_args` to those mocks to make sure that the token and url is being parsed correctly?

Reply:
I have added one check here https://github.com/elastisys/compliantkubernetes-apps/blob/rw/make-get-swift-ip-simpler/tests/unit/general/bin-update-ips-swift.bats#L179. Would you like to have more of them?

Reply:
It would be nice to also verify that the swift URL is being parsed correctly. If it's done correctly it should eventually be passed as an argument to `dig`.